L1Trigger/CSCTriggerPrimitives/src/CSCCathodeLCTProcessor.cc

```diff
@@ -905,7 +908,7 @@ bool CSCCathodeLCTProcessor::patternFinding(
                 hit_layer[this_layer] = true;
                 layers_hit++;  // determines number of layers hit
                 // add this strip in this layer to the pattern we are currently considering
-                hits_single_pattern[this_layer][strip_num] = this_strip;
+                hits_single_pattern[this_layer][strip_num] = this_strip - stagger[this_layer];
```

Comment:
... and here.

Reply:
Yes, the half-strip stagger was supposed to have been fixed in 11_1_X. But ultimately only made it into 11_2_X. See https://github.com/cms-sw/cmssw/pull/29830. This commit https://github.com/cms-sw/cmssw/pull/29830/commits/29c7161708e0bfbe8872e69e36c5dd8fcba2460b. The container `hits_single_pattern` keeps track of which comparator hits were ultimately used to build a CLCT, which you need when you want to match CLCTs to simtracks through compator digis.

Reply:
OK, thanks.